src/keylessBackup/SignInWithEmail.tsx

```diff
@@ -33,6 +33,11 @@ function SignInWithEmail({ route }: Props) {
 
   const onPressGoogle = async () => {
     setLoading(true)
+    dispatch(
```

Comment:
I moved  keylessBackupStarted here because it is the first screen that is part of both the Restore and Setup flow, so I thought it made sense for simplicity and consistency.